light-clients/ics08-wasm/src/consensus_state.rs

```diff
@@ -68,7 +70,7 @@ where
 	}
 
 	fn timestamp(&self) -> Timestamp {
-		Timestamp::from_nanoseconds(self.timestamp).expect("timestamp is valid")
```

Comment:
this is now unimplemented -- this is the line that i am erroring out when trying to use this version of the hyperspace relayer

Reply:
i think the timeouts should be re-enabled, otherwise we can't timeout packets and they will be stuck

Reply:
Re-implemented, thanks!